lca-cli/postpivot/postpivot.go

```diff
@@ -53,6 +57,7 @@ func NewPostPivot(scheme *runtime.Scheme, log *logrus.Logger, ops ops.Ops, authF
 const (
 	nodeIpFile       = "/run/nodeip-configuration/primary-ip"
 	dnsmasqOverrides = "/etc/default/sno_dnsmasq_configuration_overrides"
+	sshPublicKeyFile = "/home/core/.ssh/authorized_keys.d/recipient-keys"
```

Comment:
We can any other name, just took the one @mresvanis had. This path is where we will write ssh key for early access it will be removed at the end by MCO

Reply:
Let's call it `ib-early-access` or some other descriptive name, avoid `recipient` as it's not really an official term we use